tests/test-module.py

```diff
@@ -158,7 +166,51 @@ def test_small_result_returns_event(self):
         }
         assert result == expected_result
 
-    @patch.object(cumulus_message_adapter, 'MAX_NON_S3_PAYLOAD_SIZE', 1)
+
+    @patch('uuid.uuid4')
+    def test_configured_big_result_stored_remotely(self, uuid_mock):
+        """
+        Test remote payload is stored in S3 and return value points
+        to remote location with 'replace' key/value and correct configuration
+        """
+        event_with_ingest = {
+            'cumulus_meta': {
+                'workflow': 'testing',
+                'system_bucket': self.bucket_name
+            },
+            'meta': 'some meta object',
+            'ReplaceConfig': {
+                'Path': '$.payload',
+                'MaxSize': 1,
+                'TargetPath': '$.payload'
+            },
+        }
+        expected_create_next_event_result = {
+            'cumulus_meta': {
+                'workflow': 'testing',
+                'system_bucket': self.bucket_name
+            },
+            'meta': 'some meta object',
+            'payload': {},
```

Comment:
Defaults to an empty object because there was no payload on `event_with_ingest`?

Reply:
If this is the code setting that value

https://github.com/nasa/cumulus-message-adapter/blob/master/message_adapter/message_adapter.py#L396

Then I would expect it to be equal to the contents of the handlerResponse and not an empty object. Thus, that's probably not the code setting the value

Reply:
The code setting that is https://github.com/nasa/cumulus-message-adapter/pull/58/files#diff-31d0f5fe6944ff95b51397fc19843c63R464